packages/react-components/src/VoteValue.tsx

```diff
@@ -54,7 +59,10 @@ function VoteValue ({ accountId, autoFocus, isCouncil, onChange }: Props): React
   return (
     <InputBalance
       autoFocus={autoFocus}
-      defaultValue={accountId !== selectedId ? undefined : allBalances?.lockedBalance}
+      defaultValue={accountId !== selectedId
```

Comment:
Same as above, prefer being a bit less aggressive. (Another thing is that we don't have an easy way to unlock these atm, missing features, so it is problematic)